plugins/flatpak/gs-flatpak.c

```diff
@@ -2618,11 +2629,60 @@ gs_flatpak_refine_runtime_for_install (GsFlatpak *self,
 	return TRUE;
 }
 
+static GsApp *
+gs_flatpak_get_services_app_for_runtime (GsPlugin *plugin, GsApp *runtime)
+{
+	const gchar *runtime_id, *runtime_branch, *services_branch, *arch;
+	g_autofree gchar *services_id = NULL;
+	g_autofree gchar *source = NULL;
+
+	runtime_id = gs_app_get_id (runtime);
+	runtime_branch = gs_app_get_branch (runtime);
+
+	if ((g_strcmp0 (runtime_id, "com.endlessm.Platform") == 0 &&
+	     g_strcmp0 (runtime_branch, "eos3.1") == 0) ||
```

Comment:
Shouldn't it be also for _eos3.2_?

Reply:
I don't think that's necessary, there are no knowledge apps published that use the `com.endlessm.Platform//eos3.2` runtime (and never will be.)